app/org/sagebionetworks/bridge/dynamodb/DynamoParticipantOptionsDao.java

```diff
@@ -118,8 +116,6 @@ public OptionLookup getOptionForAllStudyParticipants(StudyIdentifier studyIdenti
         scan.addFilterCondition("studyKey", condition);
         
         List<DynamoParticipantOptions> mappings = mapper.scan(DynamoParticipantOptions.class, scan);
-        List<FailedBatch> failures = mapper.batchDelete(mappings);
-        BridgeUtils.ifFailuresThrowException(failures);
```

Comment:
Should we add a mock unit test to verify the mapper isn't being used for anything other than scan?